IviDriverPython/1.0/Spec/IviDriverPython.md

```diff
@@ -309,11 +309,12 @@ For example, consider a trigger repeated capability. Then `<RcName>` = `Trigger`
 
 ```python
 from typing import Dict
+from typing import Any, List
```

Comment:
`typing.Dict` with a capital 'D' and `typing.List` with a capital 'L' are deprecated since Python 3.9, which is itself EOL. 

You are already using `dict` with a lowercase 'd' and `list` with a lowercase 'l', e.g. `dict[str, Any]`, `list[int]`, so remove the deprecated imports.
```suggestion
from typing import Any
```